interview-questions.md

```diff
@@ -10,58 +10,62 @@ Then, research the question to expand on your answer. Even if you feel you have
 1. What is `this` in JavaScript?
 
   Your answer:
+  "This." is a syntax used to reference that an object is within itself.
 
   Researched answer:
-
+  The this. keyword in JS refers to the "owner" object of where the method was used.
```

Comment:
Yes sir. There are a few edge cases /nuances  you may want to look into as well.